packages/cmd/src/CommandProcessor.ts

```diff
@@ -842,6 +854,7 @@ export class CommandProcessor {
         this.log.info(`Command "${this.definition.name}" completed with success flag: "${json.success}"`);
         this.log.trace(`Command "${this.definition.name}" finished. Response object:\n${inspect(json, { depth: null })}`);
         response.endStream();
+        process.chdir(this.mCwd);
```

Comment:
Should this be wrapped in an `if (params.arguments.dcd)` clause, like the `process.chdir` above? I know it shouldn't matter here if we're `chdir`-ing to the original `cwd`, but IMO it seems ideal to keep `chdir` calls to a minimum.

Reply:
i changed this to set a flag if `chdir` is done and only undo it if flag is set.  does that work for you?

Reply:
Looks good 🙂